client/src/i18n/en/posting_journal.json

```diff
@@ -7,7 +7,7 @@
 "CREDIT":"Credit",
 "ROW":"Row",
 "ROWS":"Row(s)",
-"TOOLS":"Tools",
+"MENUS":"Menus",
```

Comment:
Since you have many keys "MENUS", I propose to simple make one key in `form.json` that says "MENU".

Also, in English, "MENUS" should be singular - "MENU".  Since we only have one menu, with multiple items.